CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# **Please** keep this file ordered alphabetically by directory paths.
+
+# Owners for the `.github` directory and all its subdirectories.
+
+/.github/ @aptos-labs/developer-platform @aptos-labs/security
```

Comment:
@gregnazario

Reply:
This was requested earlier by the security folks (you can see it in all the other repos).

Or are you saying I should be there too :)

Reply:
no, github says `This CODEOWNERS file contains errors`

Reply:
Added the team, now there's no error.